### PR TITLE
Fix: Cleanup root, frontend, and backend READMEs.

### PR DESCRIPTION
**Pull request by michaelwsherman**
- Organize and cleanup root README.
- Polish Frontend README.
- Polish backend README.

### CODE REVIEW

File: backend_apis/README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+This information is developers working directly with the backend APIs. See the root [README](../README.md) for information on deploying the solution (including the backend).
+
+We developed a comprehensive set of APIs, utilizing Python and FastAPI, which encapsulate all features available in the GenAI for Marketing repository. This implementation paves the way for streamlined data interactions and enhanced functionality, enabling you to deploy the solution in your environment.
 
 You are empowered to extend these APIs, intertwining them with your specific policies pertaining to security, access, and authentication to bolster a secure, seamless, and individualized interaction with the data and services available. Our offering merges practicality with customization, ensuring that your specific requirements are catered to with precision and reliability.
 
-## List of APIs developed in this repository
+## List of APIs Developed in This Repository
```
Comment (ghchinoy) [inline backend_apis/README.md:9]: lowercase "t"in "this"

**Review by ghchinoy — COMMENTED** [review]
Please review a few comments

File: README.md [hunk 2]
```diff
@@ -1,14 +1,34 @@
-# Generative AI for Marketing using Google Cloud
+# Generative AI for Marketing Using Google Cloud
 
-This repository provides a deployment guide showcasing the application of Google Cloud's Generative AI for marketing scenarios. It offers detailed, step-by-step guidance for setting up and utilizing the Generative AI tools, including examples of their use in crafting marketing materials like blog posts and social media content.
+**Resources enabling generative AI-powered marketing use cases on Google Cloud.**
 
-Additionally, supplementary Jupyter notebooks are provided to aid users in grasping the concepts explored in the demonstration.
+This repository showcases the application of Google Cloud's Generative AI to marketing scenarios. It contains detailed, step-by-step instructions to [TODO LINK]() deploy a solution implementing marketing-centric Generative AI capabilities, including capabilities to craft marketing materials like blog posts and social media content.
```
[the remark this answers is not in the document]
Comment (michaelwsherman) [reply README.md]: sorry, I had this in but didn't commit, fixed now.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
 
-The architecture of all the demos that are implemented in this application is as follows.    
+The architecture of the deployed solution:
 ![Architecture](/app/images/architecture.png "Architecture")
 
+Additionally, supplementary [TODO LINK] () Jupyter notebooks are provided to aid users in grasping the concepts explored in the solution.
```
[the remark this answers is not in the document]
Comment (michaelwsherman) [reply README.md]: sorry, I had this in but didn't commit, fixed now.

File: backend_apis/README.md (hunk 1 above)
Comment (michaelwsherman) [reply backend_apis/README.md:9]: "this" should be capitalized in a title, see https://capitalizemytitle.com/style/AP/ and pick your favorite style guide.

**Review by ghchinoy — APPROVED** [review]
LGTM

**Pull request merged**